drivers/mm/mm_drv_intel_adsp_mtl_tlb.c

```diff
@@ -401,6 +401,49 @@ int sys_mm_drv_unmap_region(void *virt, size_t size)
 	return sys_mm_drv_simple_unmap_region(va, size);
 }
 
+int sys_mm_drv_update_page_flags(void *virt, uint32_t flags)
```

Comment:
Please add comment on this function that it will clear the old access flags and only set new ones.

Reply:
This function is already documented in `include/zephyr/drivers/mm/system_mm.h`